pkg/util/labels/labels.go

```diff
@@ -54,6 +54,19 @@ func CloneAndRemoveLabel(labels map[string]string, labelKey string) map[string]s
 	return newLabels
 }
 
+// AddLabel returns a map with the given key and value added to the given map.
+func AddLabel(labels map[string]string, labelKey string, labelValue uint32) map[string]string {
```

Comment:
In general, label values are strings, so this function should probably take a string labelValue.